lib/l10n/app_en.arb

```diff
@@ -0,0 +1,37 @@
+{
+"crypto" : "Crypto",
+"walletTotalAmount" : "Total value of cryptos",
```

Comment:
Seria interessante colocar a descrição do campo, para ter uma ideia de onde está cada string.
Apenas um exemplo:
"details" : "Details",
"@details": {
     "description": "title of details screen"
}

Reply:
Corrigido

Reply:
Corrigido